README.md

```diff
@@ -21,6 +21,20 @@ The Web WorldWind [Developer's Guide](https://webworldwind.org/developers-guide)
 WorldWind's functionality. You'll  also find there links to many Web WorldWind resources, including a user guide. The 
 latest Web WorldWind release provides many simple examples showing how to use all of Web WorldWind's functionality.
 
+## Building
```

Comment:
Do we want also provide this information on Web's Overview page? Or at least a link to it?